definition/data/sheets/CaseEventToComplexTypes.json

```diff
@@ -5,5 +5,14 @@
     {"ID": "CaseDocument","CaseEventID": "ReviewedPaperResponse","CaseFieldID": "caseDocuments","ListElementCode": "receivedDatetime","EventElementLabel": "Document submitted on","EventHintText": "","FieldDisplayOrder": "","DisplayContext": "MANDATORY"},
     {"ID": "ScannedDocument","CaseEventID": "ReviewedPaperResponse","CaseFieldID": "scannedDocuments","ListElementCode": "url","EventElementLabel": "Document Link","EventHintText": "","FieldDisplayOrder": 1,"DisplayContext": "OPTIONAL"},
     {"ID": "ScannedDocument","CaseEventID": "ReviewedPaperResponse","CaseFieldID": "scannedDocuments","ListElementCode": "type","EventElementLabel": "Document Type","EventHintText": "","FieldDisplayOrder": 2,"DisplayContext": "MANDATORY"},
-    {"ID": "ScannedDocument","CaseEventID": "ReviewedPaperResponse","CaseFieldID": "scannedDocuments","ListElementCode": "subtype","EventElementLabel": "Document Subtype","EventHintText": "","FieldDisplayOrder": 3,"DisplayContext": "MANDATORY"}
-]
\ No newline at end of file
+    {"ID": "ScannedDocument","CaseEventID": "ReviewedPaperResponse","CaseFieldID": "scannedDocuments","ListElementCode": "subtype","EventElementLabel": "Document Subtype","EventHintText": "","FieldDisplayOrder": 3,"DisplayContext": "MANDATORY"},
+    {"ID": "Respondant","CaseEventID": "SuccessfulMediation","CaseFieldID": "respondents","ListElementCode": "partyName","EventElementLabel": "Defendant name","FieldDisplayOrder": "","DisplayContext": "READONLY"},
+    {"ID": "Respondant","CaseEventID": "SuccessfulMediation","CaseFieldID": "respondents","ListElementCode": "mediationSettlementReachedAt","EventElementLabel": "Mediation settlement agreed on","FieldDisplayOrder": 1,"DisplayContext": "MANDATORY"},
+    {"ID": "Respondant","CaseEventID": "SuccessfulMediation","CaseFieldID": "staffUploadedDocuments","ListElementCode": "documentName","EventElementLabel": "Document name","FieldDisplayOrder": 1,"DisplayContext": "OPTIONAL"},
+    {"ID": "Respondant","CaseEventID": "SuccessfulMediation","CaseFieldID": "staffUploadedDocuments","ListElementCode": "documentType","EventElementLabel": "Document type","FieldDisplayOrder": 1,"DisplayContext": "OPTIONAL"},
+    {"ID": "Respondant","CaseEventID": "SuccessfulMediation","CaseFieldID": "staffUploadedDocuments","ListElementCode": "documentLink","EventElementLabel": "Mediation agreement link","FieldDisplayOrder": 1,"DisplayContext": "OPTIONAL"},
```

Comment:
documentLink refers to DM link?

Reply:
when you upload via CCD, the document is first pushed to doc store and then the returned link is displayed, so yes